prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -403,7 +403,7 @@ export TLS_KEY
 setupKubeconfig
 
 export INSTALL_DIR=${TMP_DIR}
```

Comment:
you can remove this line. This was used by the Kyma-cli script for installation the binary to temporary directory.